src/domain/event/eventDetailsButtons/EventDetailsButtons.tsx

```diff
@@ -57,7 +59,7 @@ const EventDetailsButtons: React.FC<Props> = ({
         onClick={onClickLanguage}
         selectedLanguage={selectedLanguage}
       />
-      {isEditableEvent(eventData) && (
+      {isEditable && (
```

Comment:
I would use isEditableEvent helper function but change it's internal logic

Reply:
Good catch. Dunno what I was doing :D